docs/developer/overview-developer.md

```diff
@@ -1,11 +1,25 @@
 ---
 title: Developer Documentation Overview
+summary: This article is an overview of the developer documentation for the Run:ai platform.
+authors:
+    - Jason Novich
+date: 2024-Apr-10
 ---
 
 # Overview: Developer Documentation
 
 Developers can access Run:ai through various programmatic interfaces. 
 
+## API Support
+
+The endpoints and parameters specified in the following references are the ones that are officially supported by Run:ai. Endpoints and parameters that are **NOT** listed in the reference are not officially supported.
+
+Endpoints and parameters marked as deprecated means they remain operational and can be used; however, Run:ai will **NO LONGER RECOMMEND USING THEM**, **WILL NOT ADD FUNCTIONALITY TO THEM**, and **WILL NO LONGER BE SUPPORTED AFTER**
+
+Option1 : <#of versions>
+Option 2: <time period>. 
```

Comment:
@jasonnovichRunAI keeping an open comment here until we have a decision on the timeframe

Reply:
```suggestion
For details, see the [Deprecation notifications](../home/whats-new-2-17.md#deprecation-notifications).
```